### PR TITLE
fix: description for `init` command

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
fix description

**Did you add tests for your changes?**
No
**If relevant, did you update the documentation?**
Yes
**Summary**

initialize a new `project`

**Does this PR introduce a breaking change?**
No

**Other information**
No

### CODE REVIEW

File: OPTIONS.md [hunk 1]
```diff
@@ -858,7 +858,7 @@ Commands:
   configtest|t [config-path]                                                         Validate a webpack configuration.
   help|h [command] [option]                                                          Display help for commands and options.
   info|i [options]                                                                   Outputs information about your system.
-  init|c [scaffold...] [options]                                                     Initialize a new webpack configuration.
+  init|c [scaffold...] [options]                                                     Initialize a new webpack project.
```
Comment (rishabh3112) [inline OPTIONS.md:861]:
```suggestion
  init|c [generation-path] [options]                                                 Initialize a new webpack project.
```

File: packages/webpack-cli/README.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ npx webpack-cli --help verbose
   configtest|t [config-path]                 Validate a webpack configuration.
   help|h [command] [option]                  Display help for commands and options.
   info|i [options]                           Outputs information about your system.
-  init|c [scaffold...] [options]             Initialize a new webpack configuration.
+  init|c [scaffold...] [options]             Initialize a new webpack project.
```
Comment (rishabh3112) [inline packages/webpack-cli/README.md:70]:
```suggestion
  init|c [generation-path] [options]         Initialize a new webpack project.
```

**Review by rishabh3112 — CHANGES_REQUESTED** [review]
Looks good. Left a few suggestions.

**Comment by snitin315** [comment]
I wonder why help tests for alias were passing all this time, pushing a fix now.

**Comment by alexander-akait** [comment]
@snitin315 can you rebase?

**Comment by snitin315** [comment]
> @snitin315 can you rebase?

@alexander-akait Done 👍🏻

File: OPTIONS.md (hunk 1 above)
Comment (alexander-akait) [inline OPTIONS.md:861]: I think we should change signature, because was changed

Comment (snitin315) [reply OPTIONS.md]: fixed

**Review by alexander-akait — APPROVED** [review]
/cc @webpack/cli-team

**Comment by alexander-akait** [comment]
@snitin315 Can you rebase

**Pull request merged**